.cspell/cal-itp-dictionary.txt

```diff
@@ -0,0 +1,72 @@
+# Words pertaining to Cal-ITP, its partners, and the work they do
+
```

Comment:
Not sure if here or the dev list is the right place, but we should likely add "DSDL" as a known entry?

<img width="1475" height="347" alt="Image" src="https://github.com/user-attachments/assets/01d31185-f267-4dca-8c55-520ba50daf8a" />